stub.py

```diff
@@ -0,0 +1,94 @@
+"""Stub class"""
+
+from abc import abstractmethod
+from enum import Enum
+
+from pydantic.v1 import BaseModel
+
+
+class TaskType(str, Enum):
+    FDTD = "FDTD"
```

Comment:
can you add descriptive docstrings to these classes?

Reply:
modified